KS-19-02-categories

DenisGubernatorov opened the pull request:
1) Enum Category с категориями товаров и методом возвращающим описание категории 2) Класс Item19 для товара. Со свойствами для названия, ид и категории. Метод для вывода информации о товаре

DenisGubernatorov committed:
KS-19-02-categories
1) Enum Category с категориями товаров и методом возвращающим описание категории
2) Класс Item19 для товара. Со свойствами для названия, ид и категории. Метод для вывода информации о товаре

ievetrovdev commented on src/main/kotlin/lesson19/lesson19_task2.kt:
```diff
@@ -1 +1,34 @@
+    OTHER,
+    ;
+
+    fun description(): String =
```
Функции именуются в инфинитиве и должны отражать выполняемое действие

ievetrovdev commented on src/main/kotlin/lesson19/lesson19_task2.kt:
```diff
@@ -1 +1,34 @@
+) {
+    private val id = this.hashCode()
+
+    fun itemDescription() {
```
Поскольку функция выводит на экран информацию, было бы лучше назвать: `printItemInfo`

DenisGubernatorov committed:
KS-19-02-categories
1) Переименование функций  description() -> getDescription(), itemDescription() -> printItemInfo()
